core/src/main/scala/dagr/core/cmdline/parsing/ParsingUtil.scala

```diff
@@ -124,7 +95,7 @@ private[parsing] object ParsingUtil {
     }
 
     // Finally, make the map
-    classes.map(clazz => Tuple2(clazz, getAnnotation(clazz, classOf[CLPAnnotation]))).toMap
+    classes.map(clazz => Tuple2(clazz, ReflectionUtil.findJavaAnnotation(clazz, classOf[CLPAnnotation]).get)).toMap
```

Comment:
why not just put an `import ReflectionUtil._` inside this class so we don't have to write out `ReflectionUtil` each time